lsp/markdown/completion/cron.md

```diff
@@ -0,0 +1,18 @@
+Snippet for declaring a cron job.
```

Comment:
I don't think the snippet text should say it's a snippet, instead it should describe what it does and maybe what a cron job is.

Reply:
Ah I see you copied the existing text, which I must have missed in Wes's PR. Mind fixing those up too?